common/tests/acm-normal.expected.yaml

```diff
@@ -1,4 +1,15 @@
 ---
+# Source: acm/templates/provision/clusterpool.yaml
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: ManagedClusterSet
+metadata:
+  annotations:
+    cluster.open-cluster-management.io/submariner-broker-ns: edge-broker
+  name: edge
+spec:
+  clusterSelector:
```

Comment:
Same as above. Seems this way is deprecated in https://issues.redhat.com/projects/ACM/issues/ACM-1521